legion_test/legion_test/robot/utils.py

```diff
@@ -303,6 +329,25 @@ def get_current_time(time_template):
         """
         return datetime.datetime.utcnow().strftime(time_template)
 
+    @staticmethod
+    def get_future_time(offset, time_template):
+        """
+        Get templated time on `offset` seconds in future
+
+        :param offset: time offset from current time in seconds
+        :type offset: int
+        :param time_template: time template
+        :type time_template: str
+        :return: str -- time from template
+        """
+        return (datetime.datetime.utcnow() +
+                datetime.timedelta(seconds=offset)).strftime(time_template)
+
+    @staticmethod
+    def reformat_time(time_str, initial_format, target_format):

```

Comment:
I can't find an usage of this function.

Reply:
It is used in test `Get token from EDI and set expiration date`.
```${res_expiration_date} =  Reformat time    ${token["exp"]}    %a, %d %b %Y %H:%M:%S GMT    ${date_format}```